Symbology/web/Inundation/valley_bottom.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://xml.riverscapes.xyz/Symbology/web/vector.schema.json",
+  "legend": [
+    ["hsl(60, 100%, 50%)", "Valley Bottom"],

```

Comment:
There's an extra comma at the end of this line

Reply:
Thanks, fixed it